benchmarks/benchmark_functions/pl-braket-pipeline.py

```diff
@@ -62,6 +62,8 @@ def circuit(params_):
 		qml.templates.BasicEntanglerLayers(params_, wires=range(n_wires))
 		return qml.expval(qml.PauliZ(0))
 
+
+    params = qml.init.basic_entangler_layers_normal(n_layers, n_wires, mean=0, std=0.1, seed=None)
```

Comment:
```suggestion
    params = np.random.random((n_layers, n_wires))
    params = pnp.array(params, requires_grad=True)
```

I also just saw this! Let us use numpy here, the init functions are a bit outdated (we are trying to hide them until we can delete). Also, we need to convert to pnp, so that it'll work once we do not parse arguments to differentiable tensors by default.

Reply:
We need to import numpy for this change...let's close this PR and I add the change in my black PR...